.github/CODEOWNERS

```diff
@@ -707,6 +707,10 @@
 /core/templates/services/auth-backend-api.service*.ts @vojtechjelinek
 
 
+# Stale tab identification project
```

Comment:
This doesn't need its own section. I think you can just put it with the "Exploration project" group above.

Reply:
done